server/views/topics/apicache.py

```diff
@@ -478,16 +478,6 @@ def topic_platform_info():
     return mc.topicInfo()
 
 
-def topic_media_map_list(topics_id, timespans_id):
```

Comment:
Oops! I was a little overzealous in my deleting. I think this is in fact needed. Hold on...